modules/host/host_test.go

```diff
@@ -40,7 +40,7 @@ func CreateHostTester(t *testing.T) (ht *HostTester) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	w, err := wallet.New(ct.State, tp, "")
+	w, err := wallet.New(ct.State, tp, "../../host_test.wallet")
```

Comment:
do these still need to go in a top level.

But a bigger question, `CreateHostTester` is called many times. You probably need a number here tacked onto the name. `host_test.num.wallet` perhaps?